docs/cli/cmd-options.mdx

```diff
@@ -656,6 +672,18 @@ Specify a delay before the workflow starts.
 
 Backfill start time.
 
+## target-namespace
+
+Namespace in which a handler worker will be polling for Nexus tasks on.
```

Comment:
Great job & looks good to me! Just make sure all the Temporal terms are capitalized like "Worker", "Workflow", "Task Queue" and make sure every sentence is on one line. I did the Operator file as an example.